docs/faq.md

```diff
@@ -0,0 +1,84 @@
+# Frequently Asked Questions
+
+## How do I customize X on the worker group's settings?
+
+All the options that can be customized for worker groups are listed in [local.tf](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/local.tf) under `workers_group_defaults_defaults`.
+
+Please open Issues or PRs if you think something is missing.
+
+## Why are nodes not being registered?
+
+This is usually caused by a networking or endpoint configuration issue.
```

Comment:
One other problem that causes nodes not being registered is the `aws-auth` configmap is not applied. Can we just mention this here briefly?

Reply:
Good point! Added